docs/GettingStartedDocs/Contributors/NonAccMachineSGXLinuxGettingStarted.md

```diff
@@ -0,0 +1,228 @@
+
+To check if you have it installed, run the following command.
+```bash
+dpkg --list | grep az-dcap-client
+```
+
+If you don't have Azure DCAP client installed previously, please skip the content below.
+
+In most cases the Azure version of libdcap_quoteprov.so is located in /usr/lib. Check your path before changing. Here we use /usr/lib as an example.
+
+```bash
+sudo mv /usr/lib/libdcap_quoteprov.so /usr/lib/libdcap_quoteprov.so.azure
+```
+
+Otherwise the Azure one might still get used because $PATH might have /usr/lib before the path /usr/lib/x86_64-linux-gnu with the Intel version.
+
+### 3.3 Configure the qpl
+Edit the file /etc/sgx_default_qcnl.conf. To accept insecure HTTPS cert, set the option USE_SECURE_CERT to FALSE as we will use a local caching service which doesn't have a secure cert.
+
+```bash
+USE_SECURE_CERT=FALSE
+```
+
+## 4 Set up local Provisioning Certificate Caching Service (PCCS)
+### 4.1 Register an Intel developer account
+
+and subscribe to get the subscription key. This value will be used during PCCS service bring up.
+
+If you don't have an Intel account. Go to https://www.intel.com to register one by using the button at the corner.
+
+![Intel Account](./images/intel-account.png)
+
+Sign in with your Intel account and then go to
+https://api.portal.trustedservices.intel.com/provisioning-certification
+
+You shall see the screen like this:
+
+![Subscribe1](./images/subscribe1.png)
+
+Click the subscribe button then you shall see page like this.
+
+![Subscribe2](./images/subscribe2.png)
+
+Then click subscribe button again. You shall see your subscription information.
+
+![Primary Key](./images/primary-key.png)
+
+Get the value of your primary key, which will be used later.
+
+### 4.2 Install PCCS by using its Debian package
+Install nodejs and npm if you haven't
+
+```bash
+curl -sL https://deb.nodesource.com/setup_13.x | sudo -E bash
+sudo apt-get install -y nodejs
+```
+
+This [link](https://download.01.org/intel-sgx/sgx-dcap/1.6/linux/distro/ubuntuServer18.04/debian_pkgs/web/sgx-dcap-pccs/) lists all different version of pccs, please download the most recent version that matches your OS version.
+
+In this document, we use sgx-dcap-pccs_1.6.100.2-bionic1_amd64.deb as an example, run the command blow to download the package
+```bash
+cd ~
+wget https://download.01.org/intel-sgx/sgx-dcap/1.6/linux/distro/ubuntuServer18.04/debian_pkgs/web/sgx-dcap-pccs/sgx-dcap-pccs_1.6.100.2-bionic1_amd64.deb
+```
+
+Then install the package. You will be asked to finish the configuration during the installation process.
+```bash
+sudo dpkg -i sgx-dcap-pccs_1.6.100.2-bionic1_amd64.deb
+```
+![Config](./images/config.png)
+
+Recommended config:
+- HTTPS listening port: use default value.
+- Set the PCCS service to accept local connections only: use default value.
+- Set your Intel PCS API key: use the primary key of your subscription.
```

Comment:
Is PCS above correct or should it say PCCS?

Reply:
PCS: Provisioning Certification Service
PCCS: Provisioning Certification Caching Service.
The extra C stands for Caching. I will call out the full names at the first appearance of those acronyms in the doc.

Reply:
Called out the full names accordingly.